_posts/2016-05-18-dnr-continues-to-build-gis-value-with-roi-documentation.md

```diff
@@ -0,0 +1,27 @@
+
+This year we awarded our Grand Prize Award to Aaron Austin of the Division Water Resources for his Water Related Land Use Program Update project.  This project will re-invent a labor-intensive field survey task with the integration of satellite data, automated data checks, and elimination of significant travel expenses.  Just as important, this project is anticipated to reduce a 5 to 6-year process down to one year and help the division incorporate water-related land use data into their planning documents in a more timely and comprehensive fashion.  The fact that there was a cost avoidance/re-purposing of more than 7,800 staff hours and future annual saving of almost $200,000 also helped Aaron earn the top award, not to mention the attention of all DNR Directors!
+
+Our Runner-Up Award this year went to Lee Eschler of the Division of Water Rights for his Water Right Distribution Activity Update project.  Lee was able to improve this workflow by 87%, avoid 787 hours of staff time and save his division almost $20,000 per year.  In addition to these notable quantitative results, this project also developed significant qualitative benefits such as a more connected field staff with access to near real-time water right activity data, thus enabling better, more informed  decisions in the field.
+
+ <center><div class="caption"><src="{{ "/images/DNRroi2016.jpg" | prepend: site.baseurl }}" alt="Lee Eschler and Aaron Austin with DNR Executive Director Mike Styler" /><p class="caption-text">Lee Eschler and Aaron Austin with DNR Executive Director Mike Styler</p></div>
+
+While these two GIS projects are impressive, each of the GIS projects that completed a GIS ROI and Benefit Report revealed similar evidence of their heretofore hidden value.  This illumination of impact is essential for management to assign an accurate value of GIS and its role and potential in the organization.  At DNR, we have also come to learn that documenting GIS ROI can be personally beneficial to GIS staff.  It is definitely worth your time.
+
+Our 30 plus GIS projects have revealed an annual savings of approximately $440,000 to the department – this amount keeps growing with each new GIS results documentation effort.  We average a 76% workflow improvement each time GIS is incorporated into business operations.  Our GIS ROI and Benefit Report is growing in popularity and acceptance.  If you would like to give it a try, you can download the latest version of the Excel report [here](http://www.esri.com/~/media/Files/zips/news/arcuser/0316/downsizing-roi-report).
```

Comment:
what about If you would like to give it a try, you can [download the latest version of the Excel report](http://www.esri.com/~/media/Files/zips/news/arcuser/0316/downsizing-roi-report).